modules/server/templates/configs/forseti_conf_server.yaml.tpl

```diff
@@ -6,10 +6,32 @@ global:
 
 inventory:
 
+    # You must set ONLY one of root_resource_id or composite_root_resources in
+    # your configuration. Defining both will cause Forseti to exit with an
+    # error.
+
     # Root resource to start crawling from, formatted as
     # <resource_type>/<resource_id>, (e.g. "organizations/12345677890")
     root_resource_id: ${ROOT_RESOURCE_ID}
 
+    # Composite root resources: combines multiple resource roots into a single
+    # inventory, for use across all Forseti modules. Can contain one or more
+    # resources from the GCP Resource Hierarchy in any combination.
+    # https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy
+    #
+    # All resources must grant the appropriate IAM permissions to the Forseti
+    # service account before they can be included in the inventory.
+    #
+    # Forseti Explain is not supported with a composite root at this time.
+    #
+    # Resources can exist in multiple organizations.
+    #
+    #composite_root_resources:
```

Comment:
This setting will not be user-tunable; if a user hand-edits this file with a composite root resource then Terraform will replace it the next time Terraform runs. Is this desirable?

Reply:
Would prefer if this was user-tunable, but was not sure of the best way to incorporate this, as it could potentially be multiple lines and can vary from user to user. Thoughts on how to address?

Reply:
We can do this, but it'll be fairly involved - I think that we can address this in a followup PR so that forseti-security v2.12.0 can land in time.